mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -50,6 +67,129 @@ def test_brainvision_data_filters():
     assert_equal(raw.info['highpass'], 0.1)
     assert_equal(raw.info['lowpass'], 250.)
 
+    # Heterogeneous highpass in seconds (default measurement unit)
+    with warnings.catch_warnings(record=True) as w:  # event parsing
+        raw = _test_raw_reader(
+            read_raw_brainvision, vhdr_fname=vhdr_mixed_highpass_path,
+            montage=montage, eog=eog)
+
+    trigger_warning = ['parse triggers that' in str(ww.message) for ww in w]
+    lowpass_warning = ['different lowpass filters' in str(ww.message) for ww in w]
+    highpass_warning = ['different highpass filters' in str(ww.message) for ww in w]
+
+    expected_warnings = zip(trigger_warning, lowpass_warning, highpass_warning)
+
+    assert_true(all( any([trg, lp, hp]) for trg, lp, hp in expected_warnings))
```

Comment:
`any()` is no more vague than `all()` -- it's just any n-way `or`. Also, none of the other tests actually count the number of warnings ....